### PR TITLE
[RHPAM-2810] Non potential owner of task able to update task variable through REST api

### PR DESCRIPTION
**Pull request by elguardian**
added permission check to task add content

### CODE REVIEW

**Comment by gmunozfe** [comment]
For _MVELLifeCycleManagerTest_ a fix is proposed [here](https://github.com/elguardian/jbpm/pull/5)

**Comment by gmunozfe** [comment]
Same to this test https://github.com/kiegroup/jbpm/blob/master/jbpm-services/jbpm-kie-services/src/test/java/org/jbpm/kie/services/test/RuntimeDataServiceImplTest.java#L1535
`identityProvider.setName("salaboy")`

**Comment by elguardian** [comment]
Jenkins retest this please.

**Comment by elguardian** [comment]
Jenkins retest this please.

**Comment by elguardian** [comment]
@gmunozfe plz review

File: jbpm-human-task/jbpm-human-task-core/src/main/java/org/jbpm/services/task/commands/TaskContext.java [hunk 1]
```diff
@@ -153,7 +153,7 @@ public UserGroupCallback getUserGroupCallback() {
 		return (UserGroupCallback) get(EnvironmentName.TASK_USER_GROUP_CALLBACK);
 	}
 
-	private LifeCycleManager getMvelLifeCycleManager() { 
+    public LifeCycleManager getMvelLifeCycleManager() {
```
Comment (gmunozfe) [inline jbpm-human-task/jbpm-human-task-core/src/main/java/org/jbpm/services/task/commands/TaskContext.java]: I guess this change is no longer needed

**Pull request merged**